docs-source/content/userguide/cicd/why-layering-matters.md

```diff
@@ -82,9 +85,10 @@ acceptable, or it may not.
 Another option is to find a way to keep the "same" domain, that is, the same domain
 encryption key, so that we can still roll the domain and there will be no conflicts.
 
-#### Mutating the domain configuration without losing the encryption keys
+#### Mutating Domain in Image domain home configuration without losing encryption keys
 
-If we want to make a change in a lower layer without losing our domain encryption
+If we want to make a change in a lower layer in Domain in Image domains
```

Comment:
It will be easier to read if we quote "Domain in Image"

Reply:
Let's leave this up to @rosemarymarano - she's going to make an edit pass.

I think unquoted is the general style throughout the docs.

Reply:
@jshum2479 Yes, unquoted is ok here.